.github/codeql/codeql-config.yml

```
@@ -0,0 +1,2 @@
+paths-ignore:
+  - 'examples/*'
```

Can you see if this will work?  
```suggestion
  - 'examples/kotlin/*'
```
Also, let's explain why the kotlin examples produces a failure.  
Maybe include a TODO to fix this later.